### ISSUE TITLE
MSAA (IAccessible) API don't work with winforms controls in .NET 5

### ISSUE THREAD

**Issue by qmgindi** [dotnet/winforms#4100]
MSAA API don't work in .NET 5 RC1 and RC2. Still worked in .NET 5 Preview 7.

Cannot get object from point because AccessibleObjectFromPoint fails (error "incorrect parameter"). Cannot get objects in controls because AccessibleChildren gets invalid data.

**Minimal repro:**
```
var f=new Form { Text="Form" };
var b=new Button { Text="Button" }; //AccessibleObjectFromPoint fails
f.Controls.Add(b);
var k=new DataGridView { Top=30 }; k.Columns.Add("A", "A"); //AccessibleObjectFromPoint fails; AccessibleChildren gets invalid data
f.Controls.Add(k);
f.ShowDialog();
```

**Comment by RussKie** [comment]
From https://github.com/dotnet/winforms/issues/4110#issue-721848342

> AccessibleChildren returns VARIANTs with invalid vt = VT_UI4 (must be VT_I4).

**Comment by SergeySmirnov-Akvelon** [comment]
Hi @JeremyKuhne. I see that fixes for it have been merged into [5.0](https://github.com/dotnet/winforms/pull/4119) and [master](https://github.com/dotnet/winforms/pull/4117) branches. Are there any additional actions required from us or can we close this bug?

**Comment by Zheng-Li01** [comment]
Verified the issue with latest NET SDK 6.0.100-alpha.1.20557.2, the issue has been fixed that API working with winforms controls. see below screenshot.
![UpdateCoreIsue](https://user-images.githubusercontent.com/38325459/98521743-cdf56f00-22ae-11eb-9270-6a7bab0a4cd8.PNG)
